test/types/zoneddatetime.jl

```diff
@@ -423,4 +423,14 @@ using Dates: Hour, Second, UTM, @dateformat_str
         @test typemin(ZonedDateTime) <= ZonedDateTime(typemin(DateTime), utc)
         @test typemax(ZonedDateTime) >= ZonedDateTime(typemax(DateTime), utc)
     end
+
+    @testset "serialization" begin
+        zdt = ZonedDateTime(2020, 9, 1, warsaw)
+
+        b = IOBuffer()
+        serialize(Serializer(b), zdt)
+        seekstart(b)
+
+        @test deserialize(Serializer(b)) == zdt
```

Comment:
We should have a test that emulates, or just plain is, another process with a different set of globals.
This looks like it handles that right to me, but let's have tests to prove it

Reply:
I can temporarily clear the globals for the tests